pkg/metrics/metrics.jsx

```diff
@@ -812,7 +830,7 @@ class CurrentMetrics extends React.Component {
             );
         }
 
-        const rowWrapperIface = row => ({ 'data-interface': row[0] });
+        const rowWrapperIface = row => ({ 'data-interface': row[0].props?.children ?? row[0] });
```

Comment:
So that is supposed to query the interface from the `netIO` change above, but this feels quite hackish. Once everything else is ready, let's try to find a better data structure. (You already mentioned it, but let's open a thread for it)

Reply:
Ah, this is from commit a8a4ff635b1. I find this function rather confusing. Let's rename the `rowWrapper` argument to `rowProps`, and `_columns` to `columnLabels`. But I can't think of a better implementation for this either, so let's just go with this for now.